src/components/calcite-slider/calcite-slider.scss

```diff
@@ -143,19 +143,19 @@ $tick-height: 4px;
 //     top: -16px;
```

Comment:
Comment says it’s for debugging when uncommented, @paulcpederson the slider steward 👨‍✈️might know if we can ditch that.

Reply:
I think @eriklharper  added that?

Reply:
Leaving for now since this PR just updated the variable in the commented out section.

Reply:
Yeppers I did add that.  Provides nice borders around the host element so that you can make sure the sliding handles aren't going past the edges of the element.  Was super necessary while I was making enhancements to keep the slider label text from getting obscured at the edges.